sonar/modules/documents/jsonschemas/documents/document-v1.0.0_src.json

```diff
@@ -1905,5 +1968,60 @@
     "$schema",
     "pid",
     "title"
-  ]
+  ],
+  "definitions": {
+    "role": {
+      "title": "Roles",
+      "type": "array",
+      "minItems": 1,
+      "items": {
+        "title": "Role",
+        "type": "string",
+        "enum": [
+          "dgs",
+          "prt",
+          "cre",
+          "edt",
+          "ctb"
+        ],
+        "default": "cre",
+        "form": {
+          "options": [
+            {
+              "label": "contribution_role_dgs",
+              "value": "dgs"
+            },
+            {
+              "label": "contribution_role_prt",
+              "value": "prt"
+            },
+            {
+              "label": "contribution_role_cre",
+              "value": "cre"
+            },
+            {
+              "label": "contribution_role_edt",
+              "value": "edt"
+            },
+            {
+              "label": "contribution_role_ctb",
+              "value": "ctb"
+            }
+          ]
+        }
+      }
+    },
+    "controlledAffiliation": {
+      "title": "Controlled affiliations",
+      "type": "array",
```

Comment:
You do not specify the `min_items` due to editor issue, right?

Reply:
Yes, you are right